package.xml

```diff
@@ -6,13 +6,17 @@
   <description>
     A set of ROS2 message filters which take in messages and may output those messages at a later time, based on the conditions that filter needs met.
   </description>
-  <maintainer email="ethan.gao@linux.intel.com">Ethan Gao</maintainer>
```

Comment:
Since we haven't discussed this change directly with Ethan, let's not make this change for now.